src/main/resources/petriNets/engine-processes/org_group.xml

```diff
@@ -185,7 +185,8 @@
 			assignTask(initTask)
 			setData(initTask, [
 				"filter_case": ["type":"caseRef", "value": [filterAutocomplete.value], "allowedNets":["filter"]],
-				"parentId": ["type":"text", "value":useCase.stringId]
+				"parentId": ["type":"text", "value":useCase.stringId],
+				"allowed_nets": ["type": "multichoice_map", "value": filterCase.dataSet["allowed_nets"].value]
```

Comment:
Does it work on frontend if you set only values and it has no options?

Reply:
This is on developer, how he sets the option and value for this field. In filter_preference_item this field is not even editable on frontend.